.circleci/config.yml

```diff
@@ -44,53 +44,73 @@ jobs:
           command: *store_stripe_version
 
       - restore_cache:
+          name: Restore Fabric SDK
           keys:
             - fabric-sdk-cache-{{ checksum "fabric_version.txt" }}
 
       - restore_cache:
+          name: Restore Stripe SDK
           keys:
             - stripe-sdk-cache-{{ checksum "stripe_version.txt" }}
 
       - run:
-          name: Store Xcode Version
+          name: Cache Xcode Version
           command: xcodebuild -version > xcode_version.txt
 
       - restore_cache:
-          name: Restore Carthage Cache
+          name: Restore Carthage
           keys:
             - carthage-cache-v1-{{ checksum "Cartfile.resolved" }}-{{ checksum "xcode_version.txt" }}
 
       - run:
-          name: Make bootstrap
+          name: Bootstrap
```

Comment:
A lot of these names repeat throughout this file, I'd either not make the change or update it everywhere 😅